templates/frontend/objects/article_details.tpl

```diff
@@ -38,10 +38,10 @@
 			{if $article->getLocalizedCoverImage() || $issue->getLocalizedCoverImage()}
 				<div class="cover-image">
 					{if $article->getLocalizedCoverImage()}
-						<img class="img-responsive" src="{$article->getLocalizedCoverImageUrl()|escape}"{if $article->getLocalizedCoverImageAltText()} alt="{$article->getLocalizedCoverImageAltText()|escape}"{/if}>
+						<img class="img-responsive" src="{$article->getLocalizedCoverImageUrl()|escape}"{if $article->getLocalizedCoverImageAltText()} alt="{$article->getLocalizedCoverImageAltText()|escape|default:''}"{/if}>
```

Comment:
This case won't work because the `{if}` is wrapped around the `alt` attribute, so if no alt text exists the attribute will be removed. You can use the following:

```
<img class="img-responsive" src="{$article->getLocalizedCoverImageUrl()|escape}" alt="{$article->getLocalizedCoverImageAltText()|escape|default:''}">
```

The same is true for the one below in this same file. But the other changes all look good.

Reply:
thanks @NateWr, should be good now!